tools/build_env/node/Dockerfile

```diff
@@ -1,14 +1,10 @@
 # Carefully crafted, frequently updated minimal Node.js images:
 #   https://github.com/mhart/alpine-node
-FROM mhart/alpine-node:4.4.7
+FROM mhart/alpine-node:6
 LABEL description="Docker container for uProxy build environment"
 
 # Some of our NPMs require git in order to install.
 RUN apk update
```

Comment:
Yes, it's `apt-get` for Alpine Linux, a super-small distribution ideally suited for Docker images - so much so in fact that the official Docker images are moving to Alpine:
https://www.brianchristner.io/docker-is-moving-to-alpine-linux/